modules/distr-tracing-architecture.adoc

```diff
@@ -13,32 +13,40 @@
 
-** *Client* (Jaeger client, Tracer, Reporter, instrumented application, client libraries)- The {JaegerShortName} clients are language-specific implementations of the OpenTracing API. They can be used to instrument applications for distributed tracing either manually or with a variety of existing open source frameworks, such as Camel (Fuse), Spring Boot (RHOAR), MicroProfile (RHOAR/Thorntail), Wildfly (EAP), and many more, that are already integrated with OpenTracing.
+:FeatureName: The {JaegerName}
+include::snippets/deprecated-feature.adoc[]
+
+[TIP]
+====
+The {JaegerOperator} Operator link:https://access.redhat.com/solutions/7083722[will be removed] from the `redhat-operators` catalog in November 2025. Users must migrate to the {TempoOperator} and the {OTELName} for distributed tracing collection and storage.
+====
+
+** *Client* (Jaeger client, Tracer, Reporter, instrumented application, client libraries)- The {JaegerShortName} clients are language-specific implementations of the OpenTracing API. They might be used to instrument applications for distributed tracing either manually or with a variety of existing open source frameworks, such as Camel (Fuse), Spring Boot (RHOAR), MicroProfile (RHOAR/Thorntail), Wildfly (EAP), and many more, that are already integrated with OpenTracing.
 
 ** *Agent* (Jaeger agent, Server Queue, Processor Workers) - The {JaegerShortName} agent is a network daemon that listens for spans sent over User Datagram Protocol (UDP), which it batches and sends to the Collector. The agent is meant to be placed on the same host as the instrumented application. This is typically accomplished by having a sidecar in container environments such as Kubernetes.
 
 ** *Jaeger Collector* (Collector, Queue, Workers) - Similar to the Jaeger agent, the Jaeger Collector receives spans and places them in an internal queue for processing. This allows the Jaeger Collector to return immediately to the client/agent instead of waiting for the span to make its way to the storage.
 
-** *Storage* (Data Store) - Collectors require a persistent storage backend. {JaegerName} has a pluggable mechanism for span storage. {JaegerName} supports the Elasticsearch storage.
+** *Storage* (Data Store) - Collectors require a persistent storage backend. {JaegerName} has a pluggable mechanism for span storage. {JaegerName} supports the Elasticsearch??? storage.
 
 ** *Query* (Query Service) - Query is a service that retrieves traces from storage.
 
-** *Ingester* (Ingester Service) - {DTProductName} can use Apache Kafka as a buffer between the Collector and the actual Elasticsearch backing storage. Ingester is a service that reads data from Kafka and writes to the Elasticsearch storage backend.
+** *Ingester* (Ingester Service) - {DTProductName} can use Apache Kafka as a buffer between the Collector and the actual Elasticsearch??? backing storage. Ingester is a service that reads data from Kafka and writes to the Elasticsearch??? storage backend.
```

Comment:
@pavolloffay, what about these lines?

Reply:
And also the same question about line `52` below:
https://github.com/openshift/openshift-docs/pull/83995/files#diff-b159f43f7dfc6f0b1c5fc26d7f9a40863e072e3a0b11f20ae382331fd87530d4R52